parastell/invessel_build.py

```diff
@@ -266,6 +316,34 @@ def angles_to_xyz(self, toroidal_angle, poloidal_angles, s, scale):
 
         return np.array(coords) * scale
 
+    def calculate_tangents(self, toroidal_angle, poloidal_angles, s, scale):
```

Comment:
If the only difference between these is the value of `eps`, does it have to be a virtual function defined in the derived class?  Can't it be in the parent class and have this value be defined in the child class initializer?